.github/workflows/test.yml

```diff
@@ -0,0 +1,36 @@
+name: Test
+
+on:
+  schedule:
+    - cron: "0 6 * * *"
+  workflow_dispatch:
```

Comment:
could we change the model name in action?
Do we need to define `inputs` under the `workflow_dispatch` for the model names?